index.html

```diff
@@ -6,6 +6,392 @@
+			</p>
+			<p>
+			Could texts be used not only to send reminders but also to build an ongoing two-way dialogue with a trusted health partner? Would this conversation help improve medication adherence?
+			</p>
+		</article>
+
+		<!-- rotate the figure using css -->
+		<figure>
+			<img src="assets/phone.svg">
+		</figure>
+	</section>
+	<!-- end the question -->
+
+	<section class="row" id="meetFlorence">
+		<header>
+			<figure>
+				<img src="assets/florence.svg" alt="Florence is a friendly looking nurse with strawberry blonde hair">	
+			</figure>
+			<span class="subTitle">THE CONCEPT</span>
+			<h1>Meet Florence</h1>
+			<p>
+				Our explorations led to the concept of Florence — a personal companion that would help patients understand and manage their medication regimen.  A chatbot at heart, patients would converse with her through text and picture messages. 
+			</p>
+			<p>
+				If developed, a simple, familiar interface would hide her powerful intelligence. Behind the scenes, machine learning and behavioral modeling would enable her to offer a tailored, human experience based on each patient’s needs.
+			</p>
+		</header>
+
+		<!-- nest sections within the meetFlorence section -->
+		<!-- these sections will be styled as white boxes and share a common class -->
+
+		<section class="row underFlorence">
+
+		<!-- the firstHalf and secondHalf classes will both be on the same line, side-by-side -->
+
+			<section class="firstHalf">
+				<h1>SHE GETS TO KNOW YOU.</h1>
+				<p>Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
+				<p>This conversation unfolds over time, making this feel less like twenty questions and more like the start of a relationship.</p>
+			</section>
+			<section class="secondHalf">
+				<div class="grayC">
+					Hi! I'm Florence. What's your name?
+				</div>
+				<div class="blueC">
+					Mike
+				</div>
+				<div class="grayC">
+					Nice to meet you, Mike. I can help you set up your account with a few simple questions.
+				</div>
+				<div class="grayC">
+					You can answer these now or whenever you like. I'm always here!
+				</div>
+				<img src="assets/florence-avatar.svg" alt="Florence's portrait">
+				<div class="grayC">
+					First question: how old are you?
+				</div>
+				<div class="blueC">
+					65
+				</div>
+			</section>
+
+			<!-- this image must have an absolute position -->
+			<img src="assets/section1-illo.svg" alt="A steaming cup of tea">
+		</section>
+
+		<!-- note the above format works well for the rest of the chat bubbles -->
+
+		<section class="row underFlorence">
+			<section class="firstHalf">
+				<h1>SHE KEEPS TRACK OF YOUR REGIMEN.</h1>
```

Comment:
yup, `<h3>` here

There is a myth in HTML5 that "headings reset in a new section".  No screen reader supports this, so it's just that, a myth.  Your page should only have 1 `<h1>` and headings should convey the hierarchy of your whole page.